website/models.py

```diff
@@ -93,6 +95,65 @@ def save(self, *args, **kwargs):
         super(Event, self).save(*args, **kwargs)
 
 
+EventPlugin = create_plugin_base(Event)
```

Comment:
Adds extensions to the drag-drop editor in Django Admin
i.e. Rich Text, Download Button, Embeds, Image LightBox